modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/SnapshotRestoreProcess.java

```diff
@@ -1114,11 +1114,14 @@ private IgniteInternalFuture<Boolean> preload(UUID reqId) {
 
             try {
                 if (log.isInfoEnabled() && !snpAff.isEmpty()) {
-                    log.info("Trying to request partitions from remote nodes " +
-                        "[reqId=" + reqId +
-                        ", snapshot=" + opCtx0.snpName +
-                        ", map=" + snpAff.entrySet().stream().collect(Collectors.toMap(Map.Entry::getKey,
-                            e -> partitionsMapToString(e.getValue()))) + ']');
+                    for (Map.Entry<UUID, Map<Integer, Set<Integer>>> m : snpAff.entrySet()) {
+                        log.info("Trying to request partitions from remote nodes " +
+                            "[reqId=" + reqId +
+                            ", snapshot=" + opCtx0.snpName +
+                            ", consistentId=" + locNode.consistentId() +
```

Comment:
It is assumed initially, that we print corresponding remote node consistentId here.
I suggest this format for per-node message (for single remote node):
```
reqId, nodeId, consistentId, map=[[cacheId=id1, cacheName=cache1, parts=[m1,..,n1]], [cacheId=id2, cacheName=cache2, parts=[m2,..,n2]]
```

Reply:
thank you, I have changed the output by updating consistentId. Adding cacheName, groupName is tricky as we discussed it with you last week. Is it strongly necessary?